USFMToolsSharp.Renderers.Docx/DocxRenderer.cs

```diff
@@ -150,6 +150,16 @@ private void RenderMarker(Marker input, StyleConfig styles, XWPFParagraph parent
                     break;
                 case VMarker vMarker:
 
+                    // If there is no parent paragraph, then we're maybe
```

Comment:
What about putting this into the extension method? That way anything using it that extension method that was failing would be good.

Reply:
Do you mean in XWPFParagraphExtensions.createRun()?  If so, then I don't think there's anything we can do there.  DocxRenderer's `parentParagraph` is outside the scope of createRun(), so it can't "reach out" to set it.